uni/lib/main.dart

```diff
@@ -110,9 +114,12 @@ Future<void> main() async {
                   ChangeNotifierProvider(
                       create: (context) => stateProviders.homePageEditingMode),
                 ],
-                child: ChangeNotifierProvider<ThemeNotifier>(
-                  create: (_) => ThemeNotifier(savedTheme),
-                  child: const MyApp(),
+                child: ChangeNotifierProvider<LocaleNotifier>(
+                  create: (_) => LocaleNotifier(savedLocale),
+                  child: ChangeNotifierProvider<ThemeNotifier>(
+                    create: (_) => ThemeNotifier(savedTheme),
+                    child: const MyApp(),
+                  ),
```

Comment:
[MultiProvider](https://pub.dev/documentation/provider/latest/provider/MultiProvider-class.html) class was created to reduce this nested childs.
Can you try to move this two providers initialization (LocaleNotifier and ThemeNotifier) to the MultiProvider above?